engine/crates/integration-tests/tests/postgres/introspection.rs

```diff
@@ -1681,6 +1681,14 @@ fn table_with_serial_primary_key_string_unique() {
             The value is not in the given array of values
           """ nin: [String]
           not: StringSearchFilterInput
+          """
+            	The string matches the given pattern.
```

Comment:
I don't understand what's happening with the leading whitespace here.

Reply:
Wonder if the graphql-parser roundtrip is messing it up. Looks like graphql-parser didn't handle block strings properly prior to this PR: https://github.com/graphql-rust/graphql-parser/pull/75 (which is unreleased as far as I can see)